Lombiq.OrchardCoreApiClient.Tester/Program.cs

```diff
@@ -49,5 +49,20 @@ public static async Task Main()
 #pragma warning disable CA1303 // Do not pass literals as localized parameters
         Console.WriteLine("Creating and setting up the tenant succeeded.");
 #pragma warning restore CA1303 // Do not pass literals as localized parameters
+
+        var editModel = new TenantApiModel
+        {
+            Description = "Tenant edited by API Client",
+            Name = "ApiClientTenant",
+            RequestUrlPrefix = "api-client-tenant-edited",
+            RequestUrlHost = "https://orcharddojo.net/",
+            Category = "API Client - Edited Tenants",
+        };
+
+        await apiClient.OrchardCoreApi.EditAsync(editModel);
+
+#pragma warning disable CA1303 // Do not pass literals as localized parameters
```

Comment:
Since this is a second such suppression, instead of using an attribute on the whole method would be better.